### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by CalMacCQ**
I have updated the index page of the docs. I have made the following changes...

- I have updated the logo to the Quantinuum logo (and replaced mentions of CQC with Quantinuum)
- I deleted the note on the change of syntax for importing backends as this refers to old-ish versions (If this should be kept let me know)
- I modified some text and the beginning and in the extensions section
- I added the links to the public slack channel and installation troubleshooting pages.

### CODE REVIEW

File: pytket/docs/index.rst [hunk 1]
```diff
@@ -64,37 +69,14 @@ additional separate module for each. Each one of these adds either some new
...
-
-    ``from pytket.backends.ibm import AerBackend``
-
-    to
-
-    ``from pytket.extensions.qiskit import AerBackend``
-
-    because ``AerBackend`` is defined in the ``pytket-qiskit`` module.
+Extensions are separate python packages can be installed using ``pip``. The installation command is ``pip install pytket-X`` where ``X`` is the name of the extension.
+To install the ``pytket-quantinuum`` package use the following command.
```
Comment (cqc-alec) [inline pytket/docs/index.rst:73]:
```suggestion

To install the ``pytket-quantinuum`` package use the following command.
```

File: pytket/docs/index.rst [hunk 2]
```diff
@@ -114,9 +96,10 @@ User Support
...
 
+- We have a slack channel for community discussion and support. You can join by following `this link <https://tketusers.slack.com/join/shared_invite/zt-18qmsamj9-UqQFVdkRzxnXCcKtcarLRA#/shared-invite/email>`_
 - You can join the `tket-users mailing list <https://list.cambridgequantum.com/cgi-bin/mailman/listinfo/tket-users>`_. If you have questions or ideas and wishes for new features you can send an email to the list and ask for help. You can also join the list to get the newest information and get in contact with other users of tket.
 - Write an email to tket-support@cambridgequantum.com and ask for help with your problem.
-- You can write a bug report on the `CQC github <https://github.com/CQCL/pytket/issues>`_ with details of the problem and we will pick that up. You can also have a look on that page so see if your problem has already been reported by someone else.
+- You can write a bug report on `github <https://github.com/CQCL/pytket/issues>`_ with details of the problem and we will pick that up. You can also have a look on that page so see if your problem has already been reported by someone else.
```
Comment (cqc-alec) [inline pytket/docs/index.rst:102]:
```suggestion
- You can write a bug report on `github <https://github.com/CQCL/tket/issues>`_ with details of the problem and we will pick that up. You can also have a look on that page to see if your problem has already been reported by someone else.
```

(I think we should encourage bug reports to go to the tket repo.)

File: pytket/docs/index.rst [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
 
-.. image:: CQCLogo.png
-   :width: 120px
+.. image:: Quantinuum_logo.png
+   :width: 200px
    :align: right
 
-``pytket`` is a python module for interfacing with `CQC`_ tket, a set of quantum
-programming tools. We currently support circuits and device architectures from
+``pytket`` is a python module for interfacing with tket, a quantum computing toolkit and optimsing compiler developed by `Quantinuum <https://www.quantinuum.com/>`_  . We currently support circuits and device architectures from
```
Comment (cqc-alec) [inline pytket/docs/index.rst:8]: Do you need the URL here since it is given at the bottom of the page?

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: pytket/docs/index.rst (hunk 2 above)
Comment (CalMacCQ) [reply pytket/docs/index.rst]: Thanks Alec, I have made the suggested changes.

File: pytket/docs/index.rst [hunk 4]
```diff
@@ -1,19 +1,20 @@
...
 
-.. image:: CQCLogo.png
-   :width: 120px
+.. image:: Quantinuum_logo.png
+   :width: 200px
    :align: right
 
-``pytket`` is a python module for interfacing with `CQC`_ tket, a set of quantum
-programming tools. We currently support circuits and device architectures from
+``pytket`` is a python module for interfacing with tket, a quantum computing toolkit and optimising compiler developed by Quantinuum. We currently support circuits and device architectures from
```
Comment (cqc-alec) [inline pytket/docs/index.rst:8]:
```suggestion
``pytket`` is a python module for interfacing with tket, a quantum computing toolkit and optimising compiler developed by `Quantinuum`_. We currently support circuits and device architectures from
```

I think you need to do this to link it to the URL below?

File: pytket/docs/index.rst [hunk 5]
```diff
@@ -25,26 +26,30 @@ Windows. To install, run
...
 
     from pytket.extensions.qiskit import AerBackend
-    b = AerBackend()                            # connect to the backend
-    compiled = b.get_compiled_circuit(c)        # compile the circuit to satisfy the backend's requirements
-    handle = b.process_circuit(compiled, 100)   # submit the job to run the circuit 100 times
-    counts = b.get_result(handle).get_counts()  # retrieve and summarise the results
+
+    backend = AerBackend()                            # connect to the backend
+    compiled_circ = b.get_compiled_circuit(circ)      # compile the circuit to satisfy the backend's requirements
+    handle = backend.process_circuit(compiled_circ, 100)   # submit the job to run the circuit 100 times
```
Comment (cqc-alec) [inline pytket/docs/index.rst:51]: Please fix alignment.

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: pytket/docs/index.rst (hunk 4 above)
Comment (CalMacCQ) [reply pytket/docs/index.rst]: Ah thanks, should be fixed now.

File: pytket/docs/index.rst (hunk 5 above)
Comment (CalMacCQ) [reply pytket/docs/index.rst:51]: Thanks - forgot to fix this earlier. Its done now.

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**